test/cpp/aot_inductor/test.cpp

```diff
@@ -55,46 +55,36 @@ TEST(AotInductorTest, BasicTest) {
       1 /*num_models*/,
       false /*is_cpu*/,
       nullptr /*cubin_dir*/));
-  const int64_t* max_output_sizes;
-  int64_t max_output_dim;
-  AOTI_RUNTIME_ERROR_CODE_CHECK(AOTInductorModelContainerGetMaxOutputShape(
-      container_handle, 0 /*output_idx*/, &max_output_sizes, &max_output_dim));
 
-  c10::IntArrayRef array_size(max_output_sizes, max_output_dim);
-  torch::Tensor output_tensor =
-      at::zeros(array_size, at::dtype(at::kFloat).device(at::kCUDA));
-  std::vector<torch::Tensor> outputs;
-  outputs.push_back(output_tensor);
+  auto input_handles =
+      torch::aot_inductor::unsafe_alloc_new_handles_from_tensors(input_tensors);
```

Comment:
Hmm, do we need to change the var name at line 48 to ```input_tensors```?

Reply:
Good catch. I should have run this cpp test locally.